deliverables/README.md

```diff
@@ -31,38 +36,154 @@ What is the role of Javascript in a web page?
 
 #### meta
 Why do we separate HTML and CSS?  
+They are separated for organization (meaning) and clarity (presentation). 
 
 #### HTML
 What are classes and IDs (and how are they different)?  
+In the CSS, a class selector is a name preceded by a full stop (“.”) and an ID selector is a name preceded by a hash character (“#”).
```

Comment:
This is the most technical, least ambiguous and hence best answer I've seen yet.  Props for mentioning actual syntax.